### PR TITLE
ALP without daemonset redeploy

### PR DESCRIPTION
**Pull request by spikecurtis**
Signed-off-by: Spike Curtis <spike@tigera.io>

## Description

Reconfigures the way we install & enable ALP.

- Merge the default and ALP variants of `calico-node` so that we always install the FlexVolume driver and volumes
- Remove the Policy Sync API path from environment variable on `calico-node`. Instead, instruct operators to set it using FelixConfiguration.

## Todos

- [ ] Tests
- [x] Documentation
- [x] Release note

## Release Note

```release-note
Includes App Layer Policy drivers & volumes in `calico-node` but leaves disabled by default.
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit d52266ae3fd247ac4b9b4e4aaf23fa2ddb2d938f

https://deploy-preview-2633--calico.netlify.com

**Comment by spikecurtis** [comment]
Once this is merged we will have to merge https://github.com/tigera/wavetank-jobs/pull/582 to fix E2Es that depended on the old install method.

File: master/getting-started/kubernetes/installation/app-layer-policy.md [hunk 1]
```diff
@@ -15,50 +15,24 @@ operate.
...
-
-  ```bash
-  kubectl apply -f {{site.url}}/{{page.version}}/manifests/alp/node-policy-only.yaml
-  ```
-
-	> **Note**: You can also
-	> [view the manifest in your browser]({{site.url}}/{{page.version}}/manifests/alp/node-policy-only.yaml){:target="_blank"}.
-	{: .alert .alert-info}
+```bash
+calicoctl get felixconfiguration default -o yaml | \
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/app-layer-policy.md]: ouch.

Can we use `--export` to remove the need for the timestamp etc in this command?

Comment (spikecurtis) [reply master/getting-started/kubernetes/installation/app-layer-policy.md]: Oh, sweet. Didn't know about that one. Have fixed.

**Pull request merged**